unicodetools/src/main/resources/org/unicode/props/ExtraPropertyAliases.txt

```diff
@@ -150,6 +151,13 @@ cjkJoyoKanji ; kJoyoKanji
 cjkKoreanEducationHanja ; kKoreanEducationHanja
 cjkKoreanName ; kKoreanName
 cjkTGH ; kTGH
+cjkRSTUnicode ; kRSTUnicode
+cjkReading ; kReading
+cjkSrc_NushuDuben ; kSrc_NushuDuben
+cjkTGT_MergedSrc ; kTGT_MergedSrc
```

Comment:
Please revert this change: Tangut and Nüshu are not CJK, they should not have a cjk alias.

The name kReading is unfortunate (since this is really Nüshu-specific), but it is what it is.
I guess you should add the comment that I should have added saying that these are the fields from the Tangut and Nüshu sources files.

Reply:
Fixed.